src/desktop/apps/experimental-app-shell/auction/bidderRegistrationMiddleware.ts

```diff
@@ -0,0 +1,30 @@
+export const bidderRegistrationMiddleware = (req, res, next) => {
+  const pageParts = req.path.split("/")
+  const pageType = pageParts[1]
+  console.log(req.params)
+
+  if (
+    pageType === "auction-registration" ||
+    pageType === "auction-registration2"
+  ) {
+    /**
+     * If this request is from a registration modal (trying to create a bidder),
+     * defer to the auction_support app router.
+     *
+     * @see Force: src/desktop/apps/auction_support/index.coffee
+     */
+    if (req.query["accepted-conditions"] === "true") {
+      const newUrl = req.url.replace(
+        "/auction-registration/",
+        "/auction-registration-modal/"
+      )
```

Comment:
Is this `src/desktop/apps/experimental-app-shell/auction/bidderRegistrationMiddleware.ts` file intended to be temporary until it becomes _the_ app shell? Just wondering if this logic is going to be long-lived or eventually move/go away. Not a blocker.

Reply:
Once the app shell is launched post A/B test and ENV var is removed this will be perminent.